src/Symfony/Component/Mailer/Bridge/Mailjet/Transport/MailjetApiTransport.php

```diff
@@ -104,7 +104,7 @@ protected function doSendApi(SentMessage $sentMessage, Email $email, Envelope $e
         return $response;
     }
 
-    private function getPayload(Email $email, Envelope $envelope): array
+    public function getPayload(Email $email, Envelope $envelope): array
     {
         $html = $email->getHtmlBody();
         if (null !== $html && \is_resource($html)) {
```

Comment:
Does a public method come after a protected method? Could you move it before the doSendApi method?

Reply:
Fabpot is not complaining. I'll keep it as it is for easier diff until maintainers ask this to be moved.